Skype/WebSDK/docs/PTAuthPassword.md

```diff
@@ -0,0 +1,23 @@
+
+# Authentication using Username and Password
+
+ _**Applies to:** Skype for Business 2015_
```

Comment:
This doc is very incomplete. The obvious question is "a username and password to what?" Some guy tried doing this with his gmail credentials. We should at least link to some external resource here. Tbh I don't even know how to use this auth method.